lumigator/python/mzai/backend/backend/repositories/jobs.py

```diff
@@ -10,14 +10,19 @@ class JobRepository(BaseRepository[JobRecord]):
     def __init__(self, session: Session):
         super().__init__(JobRecord, session)
 
+    def list_by_job_type(self, job_type: str, skip: int, limit: int) -> list[JobRecord] | None:
+        return (
+            self.session.query(JobRecord)
+            .where(JobRecord.job_type == job_type)
+            .offset(skip)
+            .limit(limit)
+            .all()
+        )
+
 
 class JobResultRepository(BaseRepository[JobResultRecord]):
     def __init__(self, session: Session):
         super().__init__(JobResultRecord, session)
 
     def get_by_job_id(self, job_id: UUID) -> JobResultRecord | None:
-        return (
-            self.session.query(JobResultRecord)
-            .where(JobResultRecord.job_id == job_id)
-            .first()
-        )
+        return self.session.query(JobResultRecord).where(JobResultRecord.job_id == job_id).first()
```

Comment:
do we need to change this?

Reply:
Hum, I'll check it out. It may have been the formatter :-S

Reply:
Dunno.... :-/ Reverted to the original state anyway.